File: wincpp/lib/Person.cpp
#include <iostream>
#include "Person.h"
#include "Abc.cpp"

using namespace std;

Person::Person(){	
	Person::i=100;
}

void hello(){
	cout<<"fuxk"<<endl;
}

void Person::say(){
	hello();
	cout<<"i = "<<i<<endl;
}

File: wincpp/cpplearn/Person.java
public class Person{
	
	public Person(){
		say();
	}
	
	public void say(){
		System.out.println("Person.say... ");
	}
	
	public static void main(String[] args){		
		new Student();  // 打印Student.say
	}
}

class Student extends Person{
	public Student(){
		super();
	}
	public void say(){
		System.out.println("Student.say...");
	}
}

